Applications/JupyterHub.qmd

```diff
@@ -84,11 +79,9 @@ You can install additional Python packages if necessary. This can be done both f
 
 ## Markdown and Rich Text
 
-In JupyterLab, you can create Markdown and rich text content either in a separate file or directly within Jupyter notebooks. 
-
 If you prefer to work with a separate file, you can create a new Markdown (.md) or text (.txt) from the launcher or right click on the side bar and New file. Open the file and start writing your Markdown content using the Markdown syntax. Once you've written your content, you can save the file and preview it by double-clicking on it in the file browser, which will open a preview pane showing the rendered Markdown.
 
-Alternatively, you can include Markdown and rich text directly within Jupyter Notebooks. In a notebook, you can create a new cell and change its type to "Markdown" using the cell type dropdown in the toolbar or by pressing  `M`. Then, you can write your Markdown content in the cell using the Markdown syntax. To preview the rendered Markdown, you can either run the cell by pressing `Shift` + `Enter`, or use the "Preview" option from the cell's context menu to see the rendered content without executing the cell's code.
+Alternatively, you can include Markdown and rich text directly within Notebooks. In a notebook, you can create a new cell and change its type to "Markdown" using the cell type dropdown in the toolbar or by pressing  `M`. Then, you can write your Markdown content in the cell using the Markdown syntax. To preview the rendered Markdown, you can either run the cell by pressing `Shift` + `Enter`, or use the "Preview" option from the cell's context menu to see the rendered content without executing the cell's code.
```

Comment:
```suggestion
Alternatively, you can include rich text in Markdown format directly within notebooks: create a new cell and change its type to "Markdown" using the cell type dropdown in the toolbar or by pressing `M`. Then, you can write your Markdown content in the cell using the Markdown syntax. To preview the rendered Markdown, you can either run the cell by pressing `Shift` + `Enter`, or use the "Preview" option from the cell's context menu to see the rendered content without executing the cell's code.
```

What is this "Preview" option? As far as I know you can just "execute" a markdown cell (shift+enter).  So the `rendered content without executing the cell's code.` also make no sense to me

Reply:
Ah, this was my mistake, It seems like I added it for the markdown document too which is possible in Jupyterlab. I removed the sentence in this context

Reply:
updated

Reply:
did you already push the commit to change the markdown "preview" stuff?

Reply:
yes, updated as mentioned in the comment below :)